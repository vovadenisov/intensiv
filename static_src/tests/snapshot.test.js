import React from 'react';
import renderer from 'react-test-renderer';
import ModalComponent from '../components/Modal.jsx';

describe('<ModalComponent />', () => {
    it('Renders ModalComponent', () => {
        const onCloseSpy = jest.fn();

        const component = renderer.create(
            <ModalComponent
                isOpen={ true }
                onClose={ onCloseSpy }
            />
        );

        const json = component.toJSON();

        expect(json).toMatchSnapshot();
    });
});
