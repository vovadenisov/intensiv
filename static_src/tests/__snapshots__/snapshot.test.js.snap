// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ModalComponent /> Renders ModalComponent 1`] = `
<div
  className="b-modal_container"
>
  <div
    className="b-modal"
  >
    <a
      onClick={[Function]}
    >
      Закрыть
    </a>
  </div>
</div>
`;
